source/security/private-endpoints.txt

```diff
@@ -98,113 +76,109 @@ connection using either the App Services UI or the Admin API.
-            on the endpoint details screen of the AWS VPC dashboard. This
-            value starts with ``vpce-``.
-
-            Once you have the **VPC Endpoint ID**, enter it in the input box.
-
-            Optionally, enter a comment that describes the purpose of the
-            endpoint. Then click :guilabel:`Next`.
-
-         .. step:: Modify Your VPC Private DNS
-
-            a. Use the AWS CLI to modify your VPC private DNS settings. This
-               ensures that your App can resolve the private endpoint's DNS name
-               to the correct IP address.
-   
-               .. code-block:: sh
-   
-                  aws ec2 modify-vpc-endpoint \
-                    --region <your-aws-deployment-region> \
-                    --vpc-endpoint-id <your-vpc-endpoint-id> \
-                    --private-dns-enabled
-            
+            need to add it to your App. 
+
+            a. Find the **VPC Endpoint ID**, either in the output of the CLI
+               command you ran in the previous step or on the endpoint details
+               screen of the AWS VPC dashboard. This value starts with ``vpce-``.
+            #. Copy your new VPC Endpoint's ID and paste it into the ``Your VPC
+               Endpoint ID`` field in the App Services UI, then click
+               :guilabel:`Next`.
             #. If this is the first Private Endpoint you are creating, you will 
                see a toggle you can use to either restrict or open all access from 
                requests outside your PrivateLink connection. By default, App 
                Services blocks all requests coming from outside your PrivateLink 
                connection. This setting applies to this endpoint and **all 
                additional endpoints you create**. Refer to :ref:`change access 
                restrictions <change_access_restrictions>` for more information.
+            #. Click :guilabel:`Finish Endpoint Creation`.
 
-            #. Click :guilabel:`Finish Endpoint Creation` to create the endpoint. 
-         
    .. tab::
       :tabid: api
 
       .. procedure::
 
-         .. step:: Create an Interface Endpoint in Your VPC
-
-            To use a Private Endpoint, you must first create a VPC
-            interface endpoint in AWS. You can do this using the
-            :aws:`create-vpc-endpoint
-            </cli/latest/reference/ec2/create-vpc-endpoint>` command in the
-            AWS CLI.
-
-            Modify the following command template with your private
-            endpoint information. Then, copy the command and run it in
-            your terminal:
-
+         .. step:: Create a VPC Interface Endpoint
+
+            a. Call the :admin-api-endpoint:`List VPC Private Endpoint Services
+               <operation/adminListPrivateEndpointServiceInfo>` endpoint.
+            #. Filter the response object to find the entry for your App's
+               deployment region and the endpoint type you want to create.
```

Comment:
Should we link to the [Cloud Deployment Regions](https://preview-mongodbkrollinsmdb.gatsbyjs.io/atlas-app-services/DOCSP-37831/apps/deployment-models-and-regions/#cloud-deployment-regions) documentation to make it easier for people to find their relevant deployment region?

Reply:
And should we specify that the endpoint type here can be one of `general` or `sync`? (Assuming that's accurate.)

Reply:
I don't think we need to specify that it can be sync or general. In the "Add a Sync Endpoint" section, it's changed to sync.